manager/integration/tests/test_cloning.py

```diff
@@ -392,7 +393,9 @@ def test_cloning_interrupted(client, core_api, pvc, pod, clone_pvc, clone_pod):
     crash_replica_processes(client, core_api, source_volume_name)
 
     # Step-7
-    wait_for_volume_faulted(client, source_volume_name)
+    wait_for_volume_status(client, source_volume_name,
```

Comment:
I feel we should update `wait_for_volume_faulted` instead, but only focus on checking the robust become faulted. 

This should impact all tests using wait_for_volume_faulted, right?

Reply:
Yes will impact test case in `test_ha.py` and `test_cloning.py`

Below are related cases
- test_ha_salvage
- test_all_replica_restore_failure
- test_salvage_auto_crash_all_replicas
- test_volume_reattach_after_engine_sigkill
- test_cloning_interrupted

Reply:
ok, so I suggest updating `wait_for_volume_faulted` and commenting out the detached check w/ a reason.